python/pyspark/pandas/tests/data_type_ops/testing_utils.py

```diff
@@ -41,7 +43,7 @@
     from pandas import BooleanDtype, StringDtype
 
 
-class TestCasesUtils:
+class OpsTestCase(ComparisonTestBase):
     """A utility holding common test cases for arithmetic operations of different data types."""
```

Comment:
Not a strong feeling about this, but how about rename this to `OpsTestCaseUtils` or refine the doc string ??

Because it says `"A utility ..."` in the doc string, so it maybe could confusion for someone.

Reply:
Yep, maybe `OpsTestBase` is more accurate in here and also change the doc string.